mrpodcaster/api/models.py

```diff
@@ -1 +1,15 @@
+from django.db import models
+from ..authentication.models import TelegramUser
 # Create your models here.
+
+
+class Podcast(models.Model):
+    name = models.CharField(max_length=200, default="Podcast")
+    description = models.CharField(max_length=200, default="Podcast Description")
+    podcastFile = models.FileField(upload_to="audio", default=None)
```

Comment:
```suggestion
    podcast_file = models.FileField(upload_to="audio", default=None)
```

Мы всё таки в питоне пишем